terraform/main.tf

```diff
@@ -1,17 +1,25 @@
-module "ASG-LB-ECS" {
-    source = "./modules/ASG-LB-ECS"
-    cluster_name = "PROVIDE ECS CLUSTER NAME"
-    security_groups = ["LIST OF SECURITY GROUPS"]
-    subnet_ids = ["LIST OF SUBNETS IDS"]
-    vpc_id = "ID OF THE VPC (module.vpc_module.vpc_id_output)"
-}
+# commenting this out
+# it seems like it ingress
+# not finished yet...
+#module "ASG-LB-ECS" {
+#    source = "./modules/ASG-LB-ECS"
+#    cluster_name = "PROVIDE ECS CLUSTER NAME"
+#    security_groups = ["LIST OF SECURITY GROUPS"]
+#    subnet_ids = ["LIST OF SUBNETS IDS"]
+#    vpc_id = "ID OF THE VPC (module.vpc_module.vpc_id_output)"
+#}
 
 module "vpc" {
   source = "./modules/vpc"
-
   region          = var.region
```

Comment:
region, azs, aws account number - you can get all of that via data resources. Just a suggestion, both ways are fine